metaflow/plugins/kfp/tests/flows/s3_sensor_with_formatter_flow.py

```diff
@@ -23,7 +23,7 @@ def formatter(path: str, flow_parameters: Dict[str, str]) -> str:
 
 
 @s3_sensor(
-    path=join("{datastore}", "{file_name_for_formatter_test}"),
+    path=join("{datastore}", "s3_sensor_key_files", "{file_name_for_formatter_test}"),
```

Comment:
these are really `s3_sensor_test_key_files` not `s3_sensor_key_files`.  
Else one may confuse that they are always expected to be there, (for example, are the elapsed time s3_sensor files)